servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -213,17 +231,92 @@ public int deleteExpiredX509CertRecords(int expiryTimeMins) {
     public boolean updateUnrefreshedCertificatesNotificationTimestamp(String lastNotifiedServer,
                                                                       long lastNotifiedTime,
                                                                       String provider) {
-        // Currently unimplemented for AWS
-        return false;
+        try {
+            List<Item> items = getUnrefreshedCertsRecords(lastNotifiedTime, provider);
+            updateLastNotified(lastNotifiedServer, lastNotifiedTime, items);
+        } catch (Exception ex) {
+            LOGGER.error("DynamoDB updateUnrefreshedCertificatesNotificationTimestamp Error: {}/{}", ex.getClass(), ex.getMessage());
+            return false;
+        }
+
+        return true;
     }
 
     @Override
     public List<X509CertRecord> getNotifyUnrefreshedCertificates(String lastNotifiedServer, long lastNotifiedTime) {
-        // Currently unimplemented for AWS
-        return new ArrayList<>();
+
+        Map<String, AttributeValue> expressionAttributeValues = new HashMap<>();
+        expressionAttributeValues.put(":lastNotifiedServerVal", new AttributeValue().withS(lastNotifiedServer));
+        expressionAttributeValues.put(":lastNotifiedTimeVal", new AttributeValue().withN(Long.toString(lastNotifiedTime)));
+
+        // Get all records that that should receive notifications.
+        ScanRequest scanRequest = new ScanRequest()
```

Comment:
Regarding the scan, please note the following:
1. Couldn't avoid the scan when getting the updated records as I must use consistent reading. Indexes only support eventual consistency so there's a good chance we'll miss records if we use it.
2. We will need to enable Scan DynamoDB for the IAM role in prod and any other environments in order for scanning to work.

Reply:
We actually make an update just moments when writing the lastNotifiedTime and server.
In InstanceCertManager.getUnrefreshedCertsNotifications we call 2 functions right after the other:
1. updateUnrefreshedCertificatesNotificationTimestamp - we quey the index and update the records with the lastNotifiedTime and server
2. Make the call to this function, getNotifyUnrefreshedCertificates to get all the records that were just updated by us.
If we update the records and don't read them with consistency then we'll miss these records and skip their notifications.

Reply:
Edit: There is actually no good reason for InstanceCertManager.getUnrefreshedCertsNotifications to make separate calls.
We can make a single call to the store connection and have it responsible for getting all records that should receive notifications. 
For JDBC we can leave the implementation as is (it will just do the calls one after the other) but for DynamoDB we can just return all the updated records from the query. I'll fix it and push